subset_vds.py

```diff
@@ -197,8 +204,7 @@ def subset_by_samples(
     Returns:
         VariantDataset: the subset vds
     """
-    subset_vds: VariantDataset
-    subset_vds = filter_samples(input_vds, subset_sample_list)
+    subset_vds: VariantDataset = filter_samples(input_vds, subset_sample_list)
```

Comment:
do you wanna check here that you keep at least one column? I've not bothered reading the script again in full, but I can't remember if you check that the sample subset is a true subset of the available samples in the VDS

Reply:
I had a check that one doesn't enter 0, or that `n_samples` is `<=` to the total, but I'll update the check to account for negative values of `n_sampels` as well.